students/tarasovAlexey/my-app/src/components/Message.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from "./MessageField.module.css";
+
+export default class Message extends React.Component {
+    static propTypes = {
+        author: PropTypes.string.isRequired,
+        text: PropTypes.string.isRequired,
+    };
+
+    render() {
+        return <div className={classes.message}> { this.props.author +": " + this.props.text }</div>
```

Comment:
```suggestion
        return <div className={classes.message}> { `${this.props.author}:${this.props.text}` }</div>
```
шаблонные строки выглядят привлекательней, чем конкатенация